tcms/locale/sl_SI/LC_MESSAGES/django.po

```diff
@@ -1362,14 +1362,6 @@ msgstr "Izberi vse / Odznači vse"
 msgid "Expand all cases"
 msgstr "Razširi vse"
 
-#: tcms/templates/run/table_caseruns.html:15
-msgid "Case Run ID"
-msgstr "ID testiranja"
```

Comment:
There is no "Execution ID" string in this po file. Better drop all modifications from .po files and redo them at the end of this PR or after this gets merged.

Reply:
Done. I suppose this way is a lot easier, but the other commits are not atomic